src/code/home/dialog.asm

```diff
@@ -83,6 +83,7 @@ OpenDialogInTable2::
 ; Input:
 ;   a: dialog index in table 0
 OpenDialog::
```

Comment:
Shouldn't we just remove the "old" label? It always kind annoyed me that `OpenDialog` only works for dialogs 0-FF

Reply:
Yes, me too. The thing is, it is still referenced by OpenDialogInTable1 and OpenDialogInTable2, but this reference could be a local label instead.

Reply:
Done, simply by getting rid of OpenDialog, and documenting what goes on for variants.